packages/libs/eda/src/lib/map/analysis/mapTypes/plugins/useOnAreaSelected.ts

```diff
@@ -6,7 +6,12 @@ import { Bounds, MarkerProps } from '@veupathdb/components/lib/map/Types';
 interface EssentialMarkerDataResponseProps {
   error: unknown;
   isFetching: boolean;
-  markerProps?: MarkerProps[]; // base type is all we need here
+  // donut and bar
+  markerProps?: MarkerProps[];
+  // bubble is different!
+  data?: {
+    markersData?: MarkerProps[];
+  };
```

Comment:
Can someone give me a recap of what's going on here? I find it very suspicious that we would need to do something like this. Why don't we just pass `markersData` in the same way from all places? Forgive me for not reading the code more carefully, this just stuck out to me.

Reply:
@dmfalke my comment, a) at https://github.com/VEuPathDB/web-monorepo/pull/937#issuecomment-1996506016 would be helpful. :)

Reply:
> Can someone give me a recap of what's going on here? I find it very suspicious that we would need to do something like this. Why don't we just pass `markersData` in the same way from all places? Forgive me for not reading the code more carefully, this just stuck out to me.

I think you were looking at an old version?